pghoard/restore.py

```diff
@@ -94,6 +97,7 @@ def target_args():
                              default=False, action="store_true")
             cmd.add_argument("--recovery-target-time", help="PostgreSQL recovery_target_time")
             cmd.add_argument("--recovery-target-xid", help="PostgreSQL recovery_target_xid")
+            cmd.add_argument("--recovery-target-action", help="PostgreSQL recovery_target_action (pause, promote or shutdown)")
```

Comment:
if you add: choices=["pause", "promote", "shutdown"], then the help text will reflect the choices automatically and it is not possible to give invalid values